### PR TITLE
#14: Updated tests to include Exasol V8/ Update to vsjdbc 12.0.0

### LINKED ISSUE #14: Updated tests to include Exasol V8/ Update to vsjdbc 12.0.0
Since 2023-06-02 for version 8.18.1 of Exasol database a Docker image is available on [Dockerhub](https://hub.docker.com/layers/exasol/docker-db/8.18.1/images/sha256-33ce379848b9c2d3c0535af46a9e23382f1d3ccf5fc8c27131ba5c3e6e19a065).

The current ticket therefore requests to update the integration tests of VSRDSH to use version 8.18.1 as latest default version.

Please note sibling-tickets for all [JDBC-based virtual schemas](https://github.com/exasol/virtual-schemas/blob/main/doc/user-guide/dialects.md).

### PR DESCRIPTION
**Pull request by pj-spoelders**
Closes #14

### CODE REVIEW

File: .github/workflows/dependencies_update.yml
Comment (kaklakariada) [inline .github/workflows/dependencies_update.yml:1]: Please mark this as generated

File: .github/workflows/release.yml
Comment (kaklakariada) [inline .github/workflows/release.yml:1]: Please mark this as generated

File: .settings/org.eclipse.jdt.apt.core.prefs
Comment (kaklakariada) [inline .settings/org.eclipse.jdt.apt.core.prefs:1]: Delete this and add it to .gitignore

**Review by kaklakariada — CHANGES_REQUESTED** [review]

File: .github/workflows/dependencies_update.yml
Comment (pj-spoelders) [reply .github/workflows/dependencies_update.yml:1]: done

Comment (pj-spoelders) [reply .github/workflows/dependencies_update.yml:1]: done

File: .github/workflows/release.yml
Comment (pj-spoelders) [reply .github/workflows/release.yml:1]: done

File: .settings/org.eclipse.jdt.apt.core.prefs
Comment (pj-spoelders) [reply .settings/org.eclipse.jdt.apt.core.prefs:1]: done

**Review by kaklakariada — APPROVED** [review]

**Pull request merged**